src/test/scala/org/broadinstitute/dsde/workbench/sam/directory/LdapDirectoryDAOSpec.scala

```diff
@@ -85,7 +85,7 @@ class LdapDirectoryDAOSpec extends FlatSpec with Matchers with TestSupport with
       dao.loadUser(user.id).unsafeRunSync()
     }
 
-    runAndWait(dao.deleteUser(user.id))
+    dao.deleteUser(user.id).unsafeRunSync()
```

Comment:
scalatest has a `Futures` mixin that lets you do `someFuture.futureValue`, which is IMO cleaner than `runAndWait`. is there a cats equivalent?

Reply:
unsafeRunSync _is_ the cats equivalent